src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -55,14 +64,72 @@
     :label               (i18n/label :t/about)
     :accessibility-label :about-tab}])
 
-(defn view
+(defn animated-header
+  [{:keys [scroll-amount title-opacity
+           type picture title description theme]}]
```

Comment:
I think it's better to destructure `type` differently to avoid clashing with Clojure's `type` function.

Reply:
changed the prop name instead